src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,867 @@
+                    tmdb_id: externalId && mediaType === "movie" ? externalId : undefined,
+                    tvdb_id: externalId && mediaType === "tv" ? externalId : undefined,
+                    media_type: mediaType
+                }
+            });
+
+            if (response.data) {
+                toast.success(response.data.message);
+                open = false;
+                resetFlow();
+            } else {
+                const errorMsg = (response.error as any)?.message || "Failed to start auto scrape";
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)